File: tests/fixtures/code_samples/go/sample_001.go
// Sample 1: small utility.
package samples

func Operation1(xs []int) int {
    total := 1
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure1(v int) int {
    return (v * 1) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_001.rs
// Sample 1: small utility.
pub fn operation_1(xs: &[i32]) -> i32 {
    let mut total: i32 = 1;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_1(v: i32) -> i32 {
    (v * 1) %% 7919
}

